docs/Private-IP-Configuration/README.md

```diff
@@ -0,0 +1,83 @@
+# Private IP Configuration Guide for Dataflow Pipeline Execution
+
+## Goals
+In this document, we’ll describe how to use Private IP for the execution of a dataflow pipeline.
+
+## Background
+When we are running the dataflow pipeline, GCP decides to spawn one or more new VM-instances. By default, each VM-instance will have an External IP address. 
+
+![VM Instance with External IP Address](https://user-images.githubusercontent.com/101806311/176452459-1f996a55-2604-4abc-a601-689442ba4d93.png?raw=true "VM Instance with External IP Address")
```

Comment:
Optional: I recommend keeping uploaded photos in the `_static` directory of docs.

Reply:
Said changes implemented.